mangaki/mangaki/migrations/0037_artistspelling.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.1 on 2016-04-09 19:45
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mangaki', '0036_artist_name'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ArtistSpelling',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('was', models.CharField(max_length=255)),
+                ('true_name', models.CharField(max_length=255)),
+            ],
```

Comment:
This should contain a migration to create a mapping ArtistSpelling -> Artist with the artist's name for each artist as well.